src/connector/src/sink/iceberg/java/src/main/java/com/risingwave/nimtable/SparkIcebergCompaction.java

```diff
@@ -0,0 +1,27 @@
+package com.risingwave.nimtable;
+
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+
+import java.util.List;
+
+public class SparkIcebergCompaction {
+    public static void main(String[] args) {
+        String warehousePath = args[0];
+        String database = args[1];
+        String table = args[2];
+        SparkSession session = SparkSession.builder()
+                .config("spark.sql.extensions", "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions")
+                .config("spark.sql.catalog.nimtable", "org.apache.iceberg.spark.SparkCatalog")
+                .config("spark.sql.catalog.nimtable.type", "hadoop")
```

Comment:
We need to use a jdbc catalog instead of hadoop. Otherwise, this compaction result can't take effect.

Reply:
Just changed to pass these spark catalog config from the rust sdk. In the future we can change to read the catalog type from the iceberg sink options and fill in the map with the jdbc catalog info.